lib/pbench/server/api/auth.py

```diff
@@ -150,20 +90,22 @@ def verify_auth(auth_token):
         except jwt.ExpiredSignatureError:
             try:
                 ActiveTokens.delete(auth_token)
-            except Exception:
+            except Exception as e:
                 Auth.logger.error(
-                    "User attempted Pbench expired token but we could not delete the expired auth token from the database. token: '{}'",
+                    "User passed expired token but we could not delete the token from the database. token: {!r}: {}",
                     auth_token,
+                    str(e),
                 )
-                return None
             Auth.logger.warning(
-                "User attempted Pbench expired token '{}', Token deleted from the database and no longer tracked",
+                "User passed expired token {!r}; token deleted from the database and no longer tracked",
                 auth_token,
             )
         except jwt.InvalidTokenError:
-            Auth.logger.warning("User attempted invalid Pbench token '{}'", auth_token)
-        except Exception:
+            Auth.logger.warning("User passed invalid token {!r}", auth_token)
```

Comment:
Similar question on having this be a "warning"...

Reply:
Indeed; that's true, but most of these are still not something that will be of any actually use in analyzing a problem, and there's no point in clogging up the log.

Reply:
See also https://github.com/distributed-system-analysis/pbench/discussions/2509.